packages/infolists/src/Components/Concerns/HasState.php

```diff
@@ -30,6 +30,13 @@ public function getStateUsing(?Closure $callback): static
         return $this;
     }
 
+    public function state(mixed $state): static
+    {
+        $this->getStateUsing(fn (Component $component) => $component->evaluate($state));
```

Comment:
getStateUsing() can be updated to accept mix, so we dont evaluate twice

Reply:
Yeah worried about that too. Updated.